North-Seattle-College/ad440-winter2021-thursday-repo
# Prototype Azure function to access Redis cache

**Description**
Resolves #135 
Create a prototype of Azure Function (running locally) that accesses Azure Redis cache and stores and pulls data from it. (Run Redis locally using Docker)

**Test Requirements**

1. Confirm Python install
    `$ python --version`

2. Install Docker
    Mac (https://docs.docker.com/docker-for-mac/install/)
    Windows (https://docs.docker.com/docker-for-windows/install/)
    
3. Install Redis
    (https://github.com/MicrosoftArchive/redis/releases/download/win-3.2.100/Redis-x64-3.2.100.msi)
    
**To run program:**
1. Pull redis image
    `$ docker pull redis`

2. Run container
    `$ docker run -dp 3000:3000 --name prototype-redis-cache redis`

3. Run program file
    `$ python prototype-redis-cache-connect.py`

Expected command line output:
![image](https://user-images.githubusercontent.com/13280802/108295114-05947280-714c-11eb-80c6-f185e76bb8a1.png)
